src/obsidian_tears_backend/middleware.mo

```diff
@@ -26,4 +28,24 @@ module Middleware {
 
     return (tokenIndex == index) and ((timestamp + fullDay) > Time.now());
   };
+
+  public func isNotOld(_id : Text, token : T.TokenWithTimestamp) : Bool {
```

Comment:
No need for it to be public, this is a private function (only being used in this file)